### DISCUSSION TITLE
Is this field necessary ?

### DISCUSSION THREAD

**Discussion by iamleson98** [saleor/saleor#11400] [Q&A] [answered]
https://github.com/saleor/saleor/blob/7ae893782a5009426f7e27a3558ab366ab2eeeb3/saleor/graphql/product/types/products.py#L284

Hello my friends, @IKarbowiak , @maarcingebala 
I see this attribute included in `ProductVariant` graphql type.
However I don't see the model type has that field nor some methods related to it.
Additionally, There is no `resolve_margin()` static method available inside class.

Did you mistakenly added this attribute or I missed something ?

Thank you!

**Comment by maarcingebala** [comment, marked as answer]
Hey @iamleson98, this field is used to calculate [the purchase cost of a product](https://docs.saleor.io/docs/3.x/developer/api-reference/objects/product-channel-listing#code-style-fontweight-normal-productchannellistingbpurchasecostbcodemoneyrange-). It doesn't need a resolver as in Graphene fields in a type that map 1:1 to a field in a model and doesn't need any custom resolving logic are resolved automatically.
